tsl/test/expected/compression_ddl.out

```diff
@@ -1855,3 +1855,280 @@ ORDER BY device_id;
 
 SET enable_seqscan = default;
 DROP TABLE compression_insert;
+-- check Chunk Append plans for partially compressed chunks
+-- F: fully compressed, P : partially compressed, U: uncompressed
+create table test_partials (time timestamptz not null, a int, b int);
```

Comment:
Nit: Could you convert the SQL statements to uppercase?

Reply:
fixed